src/components/atoms/markdown/CodeFence.tsx

```diff
@@ -36,7 +36,33 @@ const CodeFenceCopyButton = styled.button`
     color: ${({ theme }) => theme.navButtonActiveColor};
   }
 `
+const CodeFenceSaveButton = styled.button`
```

Comment:
Not sure we need extra component with identical style. Could you just rename `CodeFenceCopyButton` to `CodeFenceButton ` and use it for both buttons?

Reply:
Changed